public/locales/en/email.json

```diff
@@ -5,27 +5,31 @@
     "error": {
       "required": "The email address is required.",
       "valid": "The email address must be valid."
-    }
+    },
+    "help-message": "Enter it exactly as provided on your passport application form.",
+    "help-message-secondary": "<strong>For child applications,</strong> provide the full email address for the applicant parent or legal guardian exactly as provided on the passport application."
   },
   "given-name": {
     "error": {
-      "required": "The given name(s) of applicant is required."
+      "required": "The given name(s) is required."
     },
-    "label": "Given name(s) of applicant"
+    "label": "Given name(s), as you detailed on the application",
+    "help-message": "Enter first name(s) and any middle name(s)."
   },
   "surname": {
     "error": {
-      "required": "The surname of applicant is required."
+      "required": "The surname is required."
     },
-    "label": "Surname of applicant"
+    "label": "Surname, as you detailed on the application",
+    "help-message": "Enter last name."
   },
   "date-of-birth": {
     "error": {
-      "current": "The applicant date of birth must be in the past.",
-      "invalid": "The applicant date of birth must be valid in the following format: YYYY-MM-DD (Year-Month-Day).",
-      "required": "The applicant date of birth is required."
+      "current": "The date of birth must be in the past.",
+      "invalid": "The date of birth must be valid in the following format: YYYY-MM-DD (Year-Month-Day).",
+      "required": "Date of birth, as you detailed the application"
```

Comment:
I think they just typo'd this in the requirements, 2/3 of them say "...detailed on the application", so I'm assuming it should be "on the application" everywhere? Maybe verify with them in the standup chat.